solidity-v1/dashboard/src/css/withdraw-granted-tokens.less

```diff
@@ -0,0 +1,101 @@
+.withdraw-granted-tokens {
+  &__form_checkbox {
+  }
+
+  &__total-ready-to-release-amount {
+    display: inline;
+  }
+
+  &__grants-accordion {
+    width: 100%;
+    border: 1px solid @color-grey-30;
+    border-radius: 0.5rem;
+    margin: 0.5rem 0;
+
+    .accordion {
+      &__button {
+        border-bottom: none;
+        align-items: center;
+
+        .withdraw-granted-tokens__expand-button {
+          display: inline-block;
+          content: "";
+          height: 7px;
+          width: 7px;
+          margin-top: 5px;
+          border-bottom: 1px solid black;
+          border-right: 1px solid black;
+          transform: rotate(-135deg);
+        }
+      }
+
+      &__button[aria-expanded="true"] > .withdraw-granted-tokens__expand-button,
+      &__button[aria-selected="true"]
+        > .withdraw-granted-tokens__expand-button {
+        transform: rotate(45deg);
+        transform-origin: 100% 40%;
+        margin-top: 3px;
+      }
+
+      //&__button:after {
+      //  display: inline-block;
+      //  content: "";
+      //  height: 5px;
+      //  width: 5px;
+      //  margin-top: 5px;
+      //  border-bottom: 2px solid black;
+      //  border-right: 2px solid black;
+      //  transform: rotate(-135deg);
+      //}
+      //
+      //&__button[aria-expanded="true"]::after,
+      //&__button[aria-selected="true"]::after {
+      //  transform: rotate(45deg);
+      //  transform-origin: 100% 40%;
+      //}
```

Comment:
Can we remove it?

Reply:
Fixed in b3c8dc6dd9dca8fcaa8b49daa5d3ff13a8b4e84a